global.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we still need this?

Reply:
Don't *need* it, but it's a nice to have when using .NET CLI commands. Happy to remove if you don't want it 👍